nifi-nar-bundles/nifi-azure-bundle/nifi-azure-services-api/pom.xml

```diff
@@ -36,6 +36,16 @@
             <groupId>org.apache.nifi</groupId>
             <artifactId>nifi-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure.kusto</groupId>
+            <artifactId>kusto-data</artifactId>
+            <version>5.0.1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure.kusto</groupId>
```

Comment:
Ingest may already have data as a dep. You may want to check that

Reply:
Actually we are using IngestionProperties from kusto-ingest and KustoResultTable from kusto-data
KustoIngestionResult.java -> com.microsoft.azure.kusto.ingest.IngestionProperties.java 
KustoQueryResponse.java -> com.microsoft.azure.kusto.data.KustoResultSetTable.java